index.html

```diff
@@ -948,17 +948,17 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
         </div>
 
         <div class="field">
-          <label for="email" class="label" required>Email</label>
+          <label for="email" class="label">Email</label>
           <div class="control has-icons-left has-icons-right">
-            <input id="email" class="input is-danger" type="email" placeholder="Email input" value="hello@">
+            <input id="email" class="input is-danger" type="email" placeholder="Email input" value="test@gmail.com">
             <span class="icon is-small is-left">
               <i class="fas fa-envelope"></i>
```

Comment:
You should apply `aria-hidden` to the icons.

Reply:
added, thanks!